build/README.md

```diff
@@ -631,6 +631,14 @@ Run `terraform destroy` on your cluster.
 #### `make terraform-clean`
 Remove .terraform directory with configs as well as tfstate files.
 
+#### `make terraform-test`
+Run Golang test which emulates and verifies successful execution of next two steps:
+```
+make gcloud-terraform-cluster
+make gcloud-terraform-destroy-cluster
+```
+`make terraform-init` should be invoked first.
```

Comment:
Should terraform-init be a dependency of this target then?

Reply:
I have added this dependency, the only issue that it would cost additional prompt to auth gcloud.